web/src/settings_users.ts

```diff
@@ -454,11 +552,11 @@ export function update_user_data(user_id, new_data) {
     }
 
     if (new_data.role !== undefined) {
-        $user_row.find(".user_role").text(people.get_user_type(user_id));
+        $user_row.find(".user_role").text(people.get_user_type(user_id)!);
```

Comment:
It should never be possible for `get_user_type` to return `undefined`, because we expect `settings_config.user_role_map` to be complete. We should remove that `undefined`, adding the needed assertion there.